addons/l10n_it_edi/data/invoice_it_template.xml

```diff
@@ -187,8 +187,8 @@
                     <DataScadenzaPagamento t-esc="format_date(payment.date_maturity)"/>
                     <ImportoPagamento t-esc="format_monetary(abs(payment.price_total), currency)"/>
                     <IstitutoFinanziario t-if="partner_bank.bank_id" t-esc="format_alphanumeric(partner_bank.bank_id.name[:80])"/>
-                    <IBAN t-if="partner_bank.acc_type == 'iban'" t-esc="partner_bank.sanitized_acc_number"/>
-                    <BIC t-elif="partner_bank.acc_type == 'bank' and partner_bank.bank_id.bic" t-esc="partner_bank.bank_id.bic"/>
+                    <IBAN t-if="partner_bank.acc_type == 'iban' and partner_bank.sanitized_acc_number" t-esc="partner_bank.sanitized_acc_number[:34]"/>
```

Comment:
is not this better to fail here, because it is better to fail than give something not exact.   (but the tax agency will probably not complain)

Reply:
OK, I'm reverting the change. 👍🏻 
I think what you wanted to say: "Is it not better to fail here? It is better to fail than [...]", right?
Just to make sure I understood well.